install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -127,7 +127,7 @@ data:
   disable-endpoint-crd: "{{ .Values.disableEndpointCRD }}"
 {{- end }}
 
-{{- if hasKey .Values "identityChangeGracePeriod" }}
+{{- if .Values.identityChangeGracePeriod }}
```

Comment:
With this change we can update line 136 (can't comment directly on it since it hasn't changed) from

```yaml
identity-change-grace-period: {{ default "5s" .Values.identityChangeGracePeriod | quote }}
```

to

```yaml
identity-change-grace-period: {{ include "validateDuration" .Values.identityChangeGracePeriod | quote }}
```

The 5s default is already handled at the CLI level (see [here](https://github.com/cilium/cilium/blob/b9f72929d45ba5d410ae7181c9db6770da4948e3/pkg/defaults/defaults.go#L151-L153)).

Reply:
done